File: Stock-Client/stock-client/src/main/java/com/mariposa/demo/stockclient/StockPrice.java
package com.mariposa.demo.stockclient;

// Created by: Logan Kells
// Date: 11/26/2020
// Reference: This lombok.Data class was created using the tutorial https://blog.jetbrains.com/idea/2019/11/tutorial-reactive-spring-boot-a-rest-client-for-reactive-streams/

@lombok.Data // Data using lombok.Data
@lombok.AllArgsConstructor // via lombok.AllArgsConstructor
@lombok.NoArgsConstructor // via lombok.NoArgsConstructor
public class StockPrice {
    private String symbol;
    private Double price;
    private java.time.LocalDate time;
}

File: Stock-Service/src/main/kotlin/com/mariposa/demo/stockservice/StockPrice.kt
package com.mariposa.demo.stockservice

// Created by: Logan Kells
// Date: 11/26/2020
// Reference: This data class StockPrice was created using the tutorial https://blog.jetbrains.com/idea/2019/10/tutorial-reactive-spring-boot-a-kotlin-rest-service/

// NOTE: This is a Kotlin data class: https://kotlinlang.org/docs/reference/data-classes.html#data-classes
data class StockPrice(val symbol: String,
                      val price: Double,
                      val time: java.time.LocalDateTime)
